mozetl/search/aggregates.py

```diff
@@ -0,0 +1,193 @@
+'''Firefox Desktop Search Count Datasets
+
+This job produces derived datasets that make it easy to explore search count
+data.
+
+The search_aggregates job is used to populate an executive search dashboard.
+For more information, see Bug 1381140.
+
+The search_clients_daily job produces a dataset keyed by
+`(client_id, submission_date, search_counts.engine, search_counts.source)`.
+This allows for deeper analysis into user level search behavior.
+'''
+import click
+import logging
+import datetime
+from pyspark.sql.functions import explode, col, when, udf, sum
+from pyspark.sql.types import StringType
+from pyspark.sql import SparkSession
+
+
+DEFAULT_INPUT_BUCKET = 'telemetry-parquet'
+DEFAULT_INPUT_PREFIX = 'main_summary/v4'
+DEFAULT_SAVE_MODE = 'error'
+MAX_CLIENT_SEARCH_COUNT = 10000
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+
+def search_aggregates(main_summary):
+    return agg_search_data(
+        main_summary,
+        [
+            'country',
+            'engine',
+            'source',
+            'submission_date',
+            'country',
```

Comment:
Oops, country is listed twice in this array and causes an error when running on live data.